source/blueprints/overview.rst

```diff
@@ -68,7 +68,14 @@ Depending on which blueprint you choose to run, this Quick Start should take bet
 1. Download
```

Comment:
After the line "Depending on which blueprint you choose to run, this Quick Start should take between five and fifteen minutes to complete.", add "You can use either use the MindMeld command line interface (insert hyperlink to https://www.mindmeld.com/docs/userguide/getting_started.html#command-line-interfaces) or a Python shell to set up the blueprint".